docs/2-tutorials/2-develop/6-push-notifications.html.md

```diff
@@ -2,7 +2,164 @@
+
+As of **PhoneGap Developer App** release 1.6.2 and **PhoneGap CLI** release 6.2.0 you can now test push notification functionality without needing to setup a GCM project for Android or an Apple push certification for iOS.
+
+<div class="alert--warning">Double check to ensure you're running your device and computer on the same network before continuing. </div>
+
+### Create Default PhoneGap Push Project
+
+1. Enter the following command from your terminal:
+
+		$ phonegap create myApp --template phonegap-template-push
+
+   This will create a folder named myApp in the current path location with a default project name of Hello World and id of com.phonegap.helloworld.
+
+   You can also specify a name and identifier to ensure the project is unique but still contains the default Hello World code project by specifying them as qualified parameters as shown below:
+
+		$ phonegap create myApp --id "org.myapp.sample" --name "appSample" --template phonegap-template-push
+
+2. Verify that you see the following output in your console after you run the command:
+
+		Creating a new cordova project.
+
+3. Change into the new project directory with the cd command:
+
+		$ cd myApp/
+
+4. Check to be sure you see the following set of files and folders shown below:
+
+		config.xml      hooks        platforms    plugins        www
+
+5. `cd` into the ***www*** folder and look around at the files and subfolders in there, this is the content of your app, with the entry point being the index.html file.
+
+		$ cd www/
+
+ <div class="alert--tip">**TIP:** Details about the rest of the files and folders created in the root project  will be covered in guides further along. For now just focus on the ***www*** folder and its contents.</div>
+
+6. Type `$ phonegap serve`. You will receive the server address the app is being hosted on in the output received in the console (`192.168.1.11:3000` in this example):
+
+		$ phonegap serve
+		[phonegap] starting app server...
+		[phonegap] listening on 192.168.1.11:3000
+		[phonegap]
+		[phonegap] ctrl-c to stop the server
+		[phonegap]
+
+7. Now go to your mobile device where the PhoneGap Developer App is running, enter the server address on the main screen and tap **Connect**.
+
+  <img class="mobile-image" src="/images/dev-app-enter-add.png" alt="PhoneGap Developer App, iOS"/>
+
+
+ <div class="alert--info"> **NOTE:** Tap directly on the server address displayed in the terminal screen of the PhoneGap Developer app to change it to match yours. The value filled in by default is only a sample. </div>
+
+ You should see the connection occur followed by a success message as shown below. If you receive an error of any kind, ensure
```

Comment:
Actually when I run this the first time on my iPhone, I first get the Apple alert asking me if I approve push notifications for this app. I attached my screenshot in case you want to include it: 

![image](https://cloud.githubusercontent.com/assets/359935/14827417/4286eb3e-0bb0-11e6-9be7-9332d9e533e6.png)